tier4/{{cookiecutter.project_slug}}/.github/cookiecutter.json

```diff
@@ -17,6 +17,8 @@
     "software_type":["standalone/mobile", "standalone/iot", "standalone/desktop", "standalone/web", "standalone/backend", "standalone/other", "addon", "library", "configurationFiles"],
     "languages": "None",
     "repo_type" : ["Package", "Website", "Standards", "Libraries", "Data", "Apps", "Tools", "APIs", "Docs"],
+    "usage_type" : ["Open Source", "Government Wide Reuse", "Exempt By Law", "Exempt By National Security", "Exempt By Agency System", "Exempt By Agency Mission", "Exempt By CIO", "Exempt By Policy Date"],
```

Comment:
Could we have the `usage_type` values match the code.json values? We would like to directly follow the federal-wide schema

https://github.com/GSA/code-gov-data/blob/master/schemas/schema-2.0.0.json#L86